prime-router/docs/getting-started/README.md

```diff
@@ -144,19 +148,40 @@ A `ctrl-c` will escape the running ReportStream instance.
 
 > [!TIP]
 > The recommended way to rapidly develop on the ReportStream backend is to use `./gradlew quickRun`.
-> This command skips some long running tasks, so use with caution as it will not build the FatJar, run database related tasks, or run the tests.
+> This command skips some long-running tasks, so use with caution as it will not build the FatJar, run database related tasks, or run the tests.
 
 ## Debugging
 
 `./gradlew quickrun` or `./gradlew run` will open a debug port on your locally running ReportStream instance.
 Connect your debugger remotely to port 5005.
 For profiling use the JMX port 9090.
 
-If using IntelliJ, this can be easily configured by first opening the `primer-router` folder as it's own project. Create a Remote JVM Debug Configuration with the port set to `5005`. 
+If using IntelliJ, this can be easily configured by first opening the `primer-router` folder as its own project. Create a Remote JVM Debug Configuration with the port set to `5005`. 
 
 ## Running the frontend
 
-See the [frontend React docs](../../../frontend-react/README.md) for more information.
+See the [frontend React docs](../../../frontend-react/README.md) for more information. Once you've done that and want to log in, __you need to use the same okta credentials you use when logging into the staging or production instances of the app__. 
+
+## Push a sample message through the app 
+
+1. If you haven't already, [download Postman](https://www.postman.com/downloads/) and install it. Import [the collections in the postman folder](./postman) into postman.
+
+2. If you haven't already, ensure the app is up and running. 
+
+3. Log into the LOCAL instance of the front end app. Do this by way of the front-end app and your CDC okta credentials.
+
+4. In order to send a message, you are going to need add the "sender id" to the POST header. Click on "edit" on the "development" row. 
+
+![report_stream_organizations.png](./img/report_stream_organizations.png)
+
+5. The data you seek is under "Organization Sender Settings". The client header is the sender. Concat the org name and the sender name should get you a valid value like so `client: development.dev-elims` 
```

Comment:
Make this a sub bullet of 4? Or include it as part of 4?

Reply:
wait - why? it's two actions on two pages. step (4) has you click on the "edit" button for the "development" row. that takes you to a new page. step (5) tells you what to do on the new page.

Reply:
oh! and re: the two postman collections @arnejduranovic 

One is the one you gave me that was also posted to slack a while back 

The other is the one I got from @JFisk42

Reply:
I can't even say if there are quality concerns about the postman collection that I linked. I do think it's good to have something like that available here. So, if there's any concern about adding a couple of large postman calls that might end up entirely or partially deprecated, a good idea might be to just upload a single postman call for this use case. That would be much more maintainable.

Reply:
Changes look good, I just don't understand why we need two postman collections? `ReportStream.postman_collection.json` seems adequate to me?

Reply:
ok - happy to use that one and remove the other one. I'll also update the screen shots where necessary